lessons/3_first_interactive_component/goodBadBtnExample.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Redi JS Project</title>
+  <link rel="stylesheet" href="css/main.css">
+  <link rel="icon" href="redi.png">
+</head>
+
+<body>
+  <p>How are you?</p>
+  <button id="goodButton">Good 👍</button><button id="badButton">Bad 👎</button>
+  <p id="answer"></p>
+  <style>
+    #goodButton, #badButton {
+      margin: 0.5em;
+      padding: 0.5em 2em;
+      font-size: 20pt;
+      cursor: pointer;
+    }
+    #goodButton {
+      background-color: lightgreen;
+    }
+    #badButton {
+      padding: 0.5em 2em;
+      font-size: 20pt;
+      background-color: lightcoral;
+    }
+  </style>
+  <script>
+    let goodButtonCmp = document.getElementById("goodButton");
```

Comment:
I'm torn on `let` vs `const` here for the absolute beginner. `let` sounds kinda better when pronounced as an English sentence, "Let GoodButtonCmp equal..." but `const` may help them avoid reassignment mistakes

Reply:
Hm, I still would stick with let for now, just to make it easier. When we go further we can use proper const and let when they needed.